### DISCUSSION TITLE
绳网居然可以接生了？

### DISCUSSION THREAD

**Discussion by homteam** [share121/inter-knot#3091] [其他]
## 众所周知
绳网的储存库来自github，所以文章和评论也储存在github，而每一个文章在github里都会有一个编号，并且在绳网里访问文章时的网址是：https://www.xn--5o0anh.top/?article=[文章编号]，
综上所述我们就可以像查uid一样不断更新后面的编号就可以达到接生的效果了！<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

至于这有什么用，额...可以刷好感度？<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >

**Comment by homteam** [comment]
顺便推荐一下我的博客（应该不是广告吧...）：https://homteam.github.io
就是自己弄着玩的不用太在意（可能有bug）<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by moded-hllt** [comment]
不止可以接生，还可以考古[doge]。mc百科（我的世界mod社区）也是差不多的结构，在一些序列靠前的模组评论区还可以看到考古站台。

**Comment by zzz-Rope** [comment]
标记一下，想接生就回来看看<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
